### ISSUE TITLE
"Added non-passive event listener to a scroll-blocking 'touchstart' event" warnings

### ISSUE THREAD

**Issue by medmunds** [shoelace-style/webawesome#1321]
### Describe the bug
A handful of Web Awesome components generate this warning in Chrome developer tools on touch devices:

> \[Violation] Added non-passive event listener to a scroll-blocking 'touchstart' event. Consider marking event handler as 'passive' to make the page more responsive. See https://www.chromestatus.com/feature/5745543795965952

The warning is a false positive: these components are legitimately installing non-passive touchstart listeners. But Chrome's decision to label this a "violation" makes it seem serious (and clutters the devtools console).

Affects wa-comparison, wa-rating, wa-slider, wa-split-panel.

### To Reproduce

1. Run Chrome and open devtools
2. If your current device does not have native touch support, open the devtools device toolbar (a.k.a. mobile emulation) to turn on touch emulation (Ctrl+Shift+M)
3. Go to any of these pages:
   - https://webawesome.com/docs/components/comparison/
   - https://webawesome.com/docs/components/rating/
   - https://webawesome.com/docs/components/slider/
   - https://webawesome.com/docs/components/split-panel/
4. Examine the devtools console

Results:
```
chunk.IB44PGUJ.js:225 
[Violation] Added non-passive event listener to a scroll-blocking 'touchstart' event.
Consider marking event handler as 'passive' to make the page more responsive.
See https://www.chromestatus.com/feature/5745543795965952
```

Expected: no warnings

(If you swapped steps 2 and 3—opened the demo page first _then_ switched on devtools touch emulation—you'll need to hit reload to see the warning.)

### Demo

Web Awesome's own component documentation is an effective demo. E.g.:
https://webawesome.com/docs/components/comparison/

### Screenshots

<img width="676" height="202" alt="Image" src="https://github.com/user-attachments/assets/ecc82e4b-3ecd-4046-895d-a18c70b7903d" />

### Browser / OS
 - Web Awesome 3.0.0-beta.4
 - OS: Mac (but shouldn't matter)
 - Browser: Chrome
 - Browser version: 139.0.7258.128 (but shouldn't matter)

### Additional information

These components all call event.preventDefault() in their touchstart handlers, so cannot be marked passive. But they *can* use `passive: false` to indicate they are *intentionally* adding non-passive touchstart listeners, which avoids the Chrome warning:

- https://github.com/shoelace-style/webawesome/blob/bfdd0bd6d0f059c229204b053841d24dde48f215/packages/webawesome/src/components/comparison/comparison.ts#L48
-  https://github.com/shoelace-style/webawesome/blob/bfdd0bd6d0f059c229204b053841d24dde48f215/packages/webawesome/src/internal/drag.ts#L135 (used by slider component)
- https://github.com/shoelace-style/webawesome/blob/bfdd0bd6d0f059c229204b053841d24dde48f215/packages/webawesome/src/components/rating/rating.ts#L169
- https://github.com/shoelace-style/webawesome/blob/bfdd0bd6d0f059c229204b053841d24dde48f215/packages/webawesome/src/components/split-panel/split-panel.ts#L106

**Comment by claviska** [comment]
All of these components use the drag utility and, [after this commit](https://github.com/shoelace-style/webawesome/commit/406f3a0e81b14bb3ee040738d8ec42a7c2ebff55#diff-caee4b8afbddbc1f5dca64873fb5e26d1f92aadbf5179cd5fc23922a6e60cc30L88-L90), I can't replicate the warning.

Closing as fixed in #1337 but if you're still seeing warnings, let me know!
